packages/server-admin-ui/src/styles/_bootstrap-variables.scss

```
@@ -0,0 +1,298 @@
+$sk-plugin-config-array-item-bg-light: aliceblue;
+
+//
+// Signal K Dark Mode Colors
+//
+
+$sk-bg-dark: #222630;
+$sk-border-color-dark: #343a48;
+
+$sk-body-tertiary-bg-dark: #1e222a;
+
+$sk-primary-dark: #5b9dff;
```

Stylelint (17.14.0)

[error] Unexpected empty line before $-variable (scss/dollar-variable-empty-line-before)

(scss/dollar-variable-empty-line-before)